aviary/mission/flight_phase_builder.py

```diff
@@ -175,7 +175,7 @@ def build_phase(self, aviary_options: AviaryValues = None, phase_type=EquationsO
             rate_targets = ['dh_dr']
             rate2_targets = ['d2h_dr2']
 
-        # For multi-engine cases, we may have throttle allocation control.
+        # For multiple heterogeneous-engine cases, we may have throttle allocation control.
```

Comment:
```suggestion
        # For heterogeneous-engine cases, we may have throttle allocation control.
```

Reply:
Implied an aircraft with heterogeneous engines must have more than one type of engine (can't have single heterogeneous engine)